models/model1.go

```diff
@@ -1,28 +0,0 @@
-package models
-
-import "gorm.io/gorm"
-
-type Person struct {
-	gorm.Model
-	Name      string `json:"name"`
-	ID        uint    `json:"id" sql:"AUTO_INCREMENT" gorm:"primary_key"`
-	Gender    string `json:"gender"`
-	BloodType string `json:"bloodtype"`
```

Comment:
for json tags with more than one word, words should be separated by _
BloodType string `json:"blood_type"